modules/installation-vsphere-config-yaml.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere.adoc
+
+[id="installation-vsphere-config-yaml-{context}"]
+= Sample `install-config.yaml` file for vSphere
+
+You can customize the `install-config.yaml` file to specify more details about
+your {product-title} cluster's platform or modify the values of the required
+parameters.
+
+[source,yaml]
+----
+apiVersion: v1beta4
```

Comment:
@staebler, do we need ` networking.machineCIDR`? It's in the link that Jianlin provided but not in the [installer docs](https://github.com/openshift/installer/blob/master/docs/user/vsphere/install_upi.md#setting-up-install-config-for-installer).

Reply:
No, the user does not need to set `networking.machineCIDR`. That field is used to establish an IP address for the machines that the installer creates. Since the installer is not responsible for creating any machines in a UPI installation, the installer does not need to know the machine CIDR.

Reply:
I don't know if it is better to imply to the user that they should set the machine CIDR, even though the installer will not use it, to maintain consistency with IPI installations.

Reply:
If it's not necessary, then I don't think we need to include it. I don't know what I'd tell users to specify for the value.